src/platform/packages/shared/presentation/presentation_publishing/interfaces/fetch/fetch.ts

```diff
@@ -40,6 +48,27 @@ import {
 } from './publishes_unified_search';
 import { apiPublishesProjectRouting } from './publishes_project_routing';
 
+// TODO Avoid redefining this here, fix circular dependency with presentation-containers
```

Comment:
We will address this in a follow up - see https://github.com/elastic/kibana/issues/242789